convert/tokenizer.go

```diff
@@ -283,8 +290,8 @@ func parseVocabulary(fsys fs.FS) (*Vocabulary, error) {
 		Pattern string
 		Func    func(fs.FS) (*Vocabulary, error)
 	}{
-		{"tokenizer.model", parseSentencePiece},
 		{"tokenizer.json", parseVocabularyFromTokenizer},
```

Comment:
Does this change the behavior to now prefer `tokenizer.json` over `tokenizer.model`? I remember this having an issue with Gemma 3 conversion. Previously we'd do this by only sending tokenizer.json over `ollama create`, and I had made a change to always send both and this logic would then prefer `tokenizer.model`. This somewhat reverts it back to considering `tokenizer.json`. In any case we'd need to make sure Gemma 3 still converts ok